### PR TITLE
RU - documentation localization

### PR DESCRIPTION
**Pull request by SergeyMatrosov**
### Proposed change(s)

Localization in Russian:

https://github.com/Unity-Technologies/ml-agents/blob/master/README.md
https://github.com/Unity-Technologies/ml-agents/blob/master/docs/Installation.md
https://github.com/Unity-Technologies/ml-agents/blob/master/docs/Getting-Started.md

### Types of change(s)

- [ ] Documentation update

### CODE REVIEW

**Comment by chriselion** [comment]
Hi @SergeyMatrosov - the `pre-commit` step is failing because of trailing whitespaces in the .md files. If you don't want to update them manually, the directions to install and run `pre-commit` are [here](https://github.com/Unity-Technologies/ml-agents/blob/master/com.unity.ml-agents/CONTRIBUTING.md#continuous-integration-ci), and that should fix the lines for you. Let me know if you have any problems getting it set up.

**Comment by SergeyMatrosov** [comment]
Hi @chriselion - Guess, everything should be fine by now. :)
Thanks for instructions.

File: docs/localized/RU/README.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
+<img src="https://github.com/Unity-Technologies/ml-agents/blob/master/docs/images/image-banner.png" align="middle" width="3000"/>
+
+# Unity ML-Agents Toolkit v.1.0
```
Comment (unityjeffrey) [inline docs/localized/RU/README.md:3]: can you change this to "Unity ML-Agents Toolkit Version Release 7"

**Review by unityjeffrey — CHANGES_REQUESTED** [review]

File: docs/localized/RU/README.md (hunk 1 above)
Comment (SergeyMatrosov) [reply docs/localized/RU/README.md]: Done.

**Review by unityjeffrey — APPROVED** [review]

**Comment by unityjeffrey** [comment]
thanks @SergeyMatrosov for your contribution!

**Pull request merged**